oscap_report/scap_results_parser/description_parser.py

```diff
@@ -1,15 +1,32 @@
+import re
+
 from lxml import etree
 
 from .namespaces import NAMESPACES
 
 
 class DescriptionParser():
-    @staticmethod
-    def get_full_description(item):
-        description = item.find(".//xccdf:description", NAMESPACES)
+    def __init__(self, ref_values):
+        self.ref_values = ref_values
+
+    def performe_sub_tag(self, text):
+        parts = re.split(r"<\S+sub\s", text)
```

Comment:
Parsing XML/HTML with regexps is not a good idea in general:
https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags

Are you sure there is no way to treat contents of this tag as a proper HTML or XML document?